### PR TITLE
[14.0][MIG] - purchase_fop_shipping

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

**Comment by sbejaoui** [comment]
cc/ @rousseldenis

File: purchase_fop_shipping/views/purchase_order.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record model="ir.ui.view" id="origin_type_purchase_order_tree">
+        <field name="model">purchase.order</field>
+        <field name="type">tree</field>
+        <field name="inherit_id" ref="purchase.purchase_order_tree" />
+        <field name="arch" type="xml">
+            <xpath expr="//tree" position="attributes">
+                <attribute name="decoration-danger">fop_reached == False</attribute>
```
Comment (rousseldenis) [inline purchase_fop_shipping/views/purchase_order.xml:9]: @sbejaoui Could you check this behaviour as it seems not working on runbot:

image

image

**Review by rousseldenis — APPROVED** [review]
Code and functional review

**Pull request merged**